pages/jobs.tsx

```diff
@@ -4,11 +4,16 @@ import Link from "next/link";
 import Layout from "../components/Layout";
 import JobListings from "../components/JobListings";
 
+const pageDescription =
+  "Local job opportunities in and around the Youngstown OH area.";
```

Comment:
Maybe "Local and remote job opportunities curated by the Code Youngstown community".